backend/dal/Helpers/Extensions/ParcelExtensions.cs

```diff
@@ -158,5 +159,49 @@ public static IQueryable<Entity.Parcel> GenerateQuery(this PimsContext context,
         {
             return parcel?.Id;
         }
+
+        /// <summary>
+        /// Update parcel financials
+        /// </summary>
+        /// <param name="context"></param>
+        /// <param name="parcel"></param>
+        /// <param name="parcelEvaluations"></param>
+        /// <param name="parcelFiscals"></param>
+        public static void UpdateParcelFinancials(this PimsContext context,  Entity.Parcel parcel,
+            ICollection<Entity.ParcelEvaluation> parcelEvaluations, ICollection<Entity.ParcelFiscal> parcelFiscals)
+        {
+            foreach (var parcelEvaluation in parcelEvaluations)
+            {
+                var originalEvaluation = parcel.Evaluations
+                    .FirstOrDefault(e => e.Date == parcelEvaluation.Date && e.Key == parcelEvaluation.Key);
+
+                if (originalEvaluation == null)
+                {
+                    parcel.Evaluations.Add(parcelEvaluation);
+                }
+                else
+                {
+                    originalEvaluation.Note = parcelEvaluation.Note;
```

Comment:
I noticed in the `BuildingExtensions` you use `context.Entry(existingEvaluation).CurrentValues.SetValues(buildingEvaluation);`, but here you specify each property.  Any reason?  I know that the `SetValues` can cause unintended issues sometimes.